src/test/ui/consts/const-eval/const_fn_panic_libcore.rs

```diff
@@ -0,0 +1,13 @@
+// check-pass
+
+#![no_std]
```

Comment:
Instead of two tests, please make it one test that explicitly calls `std::panic!` and `core::panic!`, respectively.

Reply:
Would you want me to just combine this with the `const_panic_libcore` and `const_panic` tests? All 4 could just be in one file.

Reply:
Yes that seems reasonable.

Reply:
I ended up merging the `libcore` tests, but kept the `const_fn_panic` and `const_panic` tests separate, so that we have one test that fails to compile and another that succeeds.